porter/v1/cluster_control_plane.proto

```diff
@@ -1914,3 +1915,21 @@ message SharedNetworkSettingsResponse {
     AWSVpc eks_cloud_provider_network = 5;
   }
 }
+
+// ListSystemServiceStatusRequest is the request object for listing system service status in a cluster
+message ListSystemServiceStatusRequest {
+  // project_id is the id of the project this cluster is in
+  int64 project_id = 1;
+  // cluster_id is the id of the cluster
+  int64 cluster_id = 2;
+  // cloud_provider is the cloud provider this cluster is located in
+  EnumCloudProvider cloud_provider = 3;
+}
+
+// ListSystemServiceStatusResponse is the response object for listing system service status
+message ListSystemServiceStatusResponse {
+  // no_cluster_hearbeat is set to true if the cluster has not heartbeat in the last 10 minutes
+  bool no_cluster_hearbeat = 1;
```

Comment:
I would call this `responsive` or `alive`, I don't care how we check that it is good/bad (heartbeat feels like an implementation detail)